src/main/java/com/commercetools/sync/commons/utils/CustomUpdateActionUtils.java

```diff
@@ -319,6 +319,8 @@
      * value or the addition of a new one. It returns a {@link List}&lt;{@link UpdateAction}&gt; as a result.
      * If no update action is needed an empty {@link List}&lt;{@link UpdateAction}&gt; is returned.
      *
+     * <p>Note: Null value custom fields are filtered out.
```

Comment:
why do we filter out null values? Why do we skip building update actions when the new value is `null`?

Reply:
ok, now I understand why. Because the other method `buildRemovedCustomFieldsUpdateActions` takes care of that. Would only suggest:
```suggestion
     * <p>Note: Null value custom fields are filtered out. In other words, no update actions would be built for fields with null values.
```

Reply:
b378c4953ef11405bff4500c05b27e29b3cb4212